todolist.js

```diff
@@ -50,6 +57,20 @@ function addTask() {
   appendResult(taskItem);
 }
 
+// タスクの削除処理
+function deleteTask(taskId) {
+  const index = tasks.findIndex(function (t) {
+    return t.id === taskId;
+  });
```

Comment:
こちら、アロー関数を使用するとなお良いです！
```suggestion
  const index = tasks.findIndex((t) => {
    return t.id === taskId;
  });
```

jsではアロー関数で定義されることが多いので、
`function() {}`
の記述はほぼ使用されることはないと思っていただいて構いません！
他の箇所も修正してみましょう！

Reply:
JSでは無名関数ではなくアロー関数というのが使われるのですね。確認し修正致します。